sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusReceivedMessage.cs

```diff
@@ -14,21 +16,58 @@ namespace Azure.Messaging.ServiceBus
     /// </summary>
     public class ServiceBusReceivedMessage
     {
+
+        /// <summary>
+        /// Creates a new message from the specified payload.
+        /// </summary>
+        /// <param name="body">The payload of the message in bytes</param>
```

Comment:
nit: This reads a bit awkwardly to me.  It sounds like you intend `bytes` as a unit of measure rather than this being the byte-serialized representation of the data.